user/enterprise/xenial.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Xenial Build Containers for Enterprise (beta)
+layout: en_enterprise
+
+---
+
+## System Setup
+
+**Platform Requirements**: To use the Xenial build containers, the Travis CI installation must be at 2.2.6 or higher. Please be sure to [upgrade](/user/enterprise/upgrading/), if needed, before getting started.
+
+**Worker Requirements**:
+
+We recommend using AWS's `c4.2xlarge` as the instance type with at least 40gb of disk space, running Ubuntu 16.04 or later. Port 22 must be open for SSH during installation and operation.
```

Comment:
We should avoid making it seem as if only AWS is supported for Travis CI Enterprise.

We recommend using a machine with 8 vCPUs and 15 GiB of memory and at least 40 GiB of disk space. If you're using AWS that will be their `c4.2xlarge` instance type. Also, you'll want to run Ubuntu 16.04 or later.

Reply:
Makes sense to me. I edited that in this file and same also for Trusty ✅